.gitignore

```diff
@@ -77,3 +77,8 @@ spark-warehouse/
 # For R session data
 .RData
 .RHistory
+.Rhistory
```

Comment:
Hmm... Is the upper/lower case difference caused by different R versions or platforms?